src/format.html

```diff
@@ -682,6 +712,21 @@ <h4 id="sensor-data">Sensor data</h4>
           <code>V</code>
         </td>
       </tr>
+      <tr>
+        <td>
+          <code>0x4E</code>
+        </td>
+        <td>volume</td>
```

Comment:
Does it make sense to have a generic volume? Should it be more precise and specify a volume of "what"

Reply:
By making it general, people can use it for whatever they want, keeping it flexible to use. But I can add water as extra object type, as that will be a very commonly used.

Reply:
We have `gas` as a type. So I would expect `water` to be one too.

Reply:
Ok, I'll add waters in a separate PR. I've created an issue such that I don't forget.

Reply:
do you still want to include volume? Or would just the water sensor cover the current use cases

Reply:
Yes, let’s keep it. Can be useful for other liquids, beer, oil,… We already had a volume sensor with less bytes as well.